packages/primitives/radio/src/Root.tsx

```diff
@@ -4,7 +4,7 @@ import { component, startWithType, mapContext, mapDefaultProps } from 'refun'
 import { TRadioInput } from './types'
 import { RadioContext } from './context'
 
-const styles = normalizeWebStyle({
+const visibleStyles = normalizeWebStyle({
   backgroundColor: 'rgba(0, 0, 0, 0)',
   borderWidth: 0,
   minWidth: 0,
```

Comment:
Do we need cursor styling as well?

Reply:
I don't think we do, but open to hear your thoughts on this